packages/theme/src/theme.ts

```diff
@@ -61,6 +61,8 @@ export class Theme extends HTMLElement implements ThemeKindProvider {
 
     private static templateElement?: HTMLTemplateElement;
 
+    private static instances: Theme[] = [];
```

Comment:
Is this an OK way to cache all the instances? It's sort of leaky... Should I use a weak map? I like better ideas.

Reply:
You need to be able to iterate the entries, so I don't think you can use a weakset/map here.

Reply:
Probably want the theme instalces to add/remove themselves as they are connected/disconnected instead?

Reply:
I don't think you can iterate over weak maps, so I don't think that's an option.

Reply:
Makes sense. Will update the add/remove functionality.